packages/react-sdk/src/css/preset-style.ts

```diff
@@ -0,0 +1,52 @@
+import { useMemo, useState } from "react";
```

Comment:
I am wondering if all this presetStyle logic should be in SDK, seems like it should be in designer, wdyt?

Reply:
It's the same as props. The state is used for rendering in canvas and published site.

Reply:
published site is using a generated static stylesheet, btw did you check if these styles land there?
canvas - can go to canvas side of the things inside designer

Reply:
Gotcha, added utility for generated css. I guess it can be moved indeed.